app/views/dashboard/index.html.slim

```diff
@@ -1,4 +1,53 @@
-h1 Dashboard
+h1 Simple Monitoring
 .row
   .col-xs-12.discovery-control
-    a.btn.btn-primary[href="/nodes"] Choose master
+    span #{Minion.count} Nodes found
+.nodes-container
+  table.table
+    thead
+      tr
+        th
+          | Id
+        th.text-center
+          | Status
+        th
+          | Hostname
+        th
+          | Role
+        th
+          | Highstate
```

Comment:
Role and Highstate are not present in the design.
Status is the first column, Node Identifier is the second one
Missing RAM, CPU and Model columns